diff --git a/node_modules/@types/bandagedbd__bdapi/index.d.ts b/node_modules/@types/bandagedbd__bdapi/index.d.ts
index e34f6e5..f545418 100644
--- a/node_modules/@types/bandagedbd__bdapi/index.d.ts
+++ b/node_modules/@types/bandagedbd__bdapi/index.d.ts
@@ -242,7 +242,7 @@ declare namespace BdApiModule {
     /**
      * An instance of AddonAPI to access plugins.
      */
-    const Plugins: typeof AddonAPI;
+    const Plugins: AddonAPI;
 
     /**
      * Gives access to BBD's internal settings object and is therefore subject to change.
